packages/fluentui/react-northstar/src/types.ts

```diff
@@ -65,8 +65,15 @@ type ReactNode = React.ReactChild | React.ReactNodeArray | React.ReactPortal | b
 export type ShorthandValue<P extends Props> =
   | ReactNode
   | (Props<P> & { children?: P['children'] | ShorthandRenderFunction<P> });
```

Comment:
```suggestion
export type ShorthandValue<P extends Props> = ReactNode | ObjectShorthandValue<P>
```
??

Reply:
And it looks that we need a custom proptypes for this, too

Reply:
Added